docs/source/supergraphs.md

```diff
@@ -43,7 +43,27 @@ subgraphs:
       file: ./people.graphql
 ```
 
-The YAML file must specify each subgraph's public-facing URL (`routing_url`), along with the path to its schema (`schema.file`).
+In the above example, The YAML file specifies each subgraph's public-facing URL (`routing_url`), along with the path to its schema (`schema.file`).
+
+It's also possible to pull subgraphs from various sources and specify them in the YAML file. For example, here is a configuration that specifies schema using Apollo Registry refs (`subgraph`, `graphref`), introspection(`url`) and subgraph introspection (`subgraph_url`):
+
+```yaml
+subgraphs:
+  films:
+    routing_url: https://films.example.com
+    schema: 
+      file: ./films.graphql
+  people:
+    schema: 
+      subgraph_url: https://example.com/people
+  reviews:
+    schema: 
+      url: https://reviews.example.com
```

Comment:
I'll just change this to use `subgraph_url`, and make the suggestion that `reviews.example.com` is a federated endpoint that supports `{ _service { sdl } }`.

```suggestion
      subgraph_url: https://reviews.example.com
```